cmd/state/internal/cmdtree/branch.go

```diff
@@ -12,7 +12,7 @@ func newBranchCommand(prime *primer.Values) *captain.Command {
 
 	return captain.NewCommand(
 		"branch",
-		locale.Tl("branch_title", "Listing branches"),
+		locale.Tl("branch_title", "Project Branches"),
```

Comment:
Let's undo this? I don't think adding a Project suffix here is helpful, and it may in some cases be plain inaccurate (headless state).